pkg/api/server/cel2sql/interpreter.go

```diff
@@ -319,6 +321,8 @@ func (i *interpreter) interpretBinaryCallExpr(expr *exprpb.Expr) error {
 		sqlOperator = posgresqlConcatOperator
```

Comment:
Not related to your changes, but there's a typo here.

Reply:
Thanks Alan, fixed it.